### PR TITLE
EL-1736: Set up Github Action Pipeline to UAT - Part 2

### PR DESCRIPTION
**Pull request by MazOneTwoOne**
- add specific deployment for UAT so that we can clean-up the release afterwards.

### CODE REVIEW

File: .github/workflows/static-analysis.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install pre-commit
-        ls requirements/source/* | xargs pre-commit run pip-compile --files
+        ls requirements/source/* | xargs pre-commit run pip-compile --files
```
Comment (willc-work) [inline .github/workflows/static-analysis.yml:26]: i think this might qualify as an `unnecessary whitespace changes.`

File: requirements/source/requirements-testing.in [hunk 2]
```diff
@@ -1,4 +1,3 @@
 -r requirements-base.in
 pytest==7.4.4
-freezegun
-httpx
+freezegun
```
Comment (willc-work) [inline requirements/source/requirements-testing.in:3]: i know these end of file errors are only stylistic but I kind of get the ethos, would you mind adding a blank line to the end of the files where these are shown.

**Review by willc-work — APPROVED** [review]

File: .github/workflows/static-analysis.yml (hunk 1 above)
Comment (MazOneTwoOne) [reply .github/workflows/static-analysis.yml]: Whoops - added when I was testing some stuff out. Now removed.

File: requirements/source/requirements-testing.in (hunk 2 above)
Comment (MazOneTwoOne) [reply requirements/source/requirements-testing.in]: Cool, have added a blank line at the end of the `requirements-.in` files, where required.
